ansible/todo_playbook.yml

```diff
@@ -0,0 +1,69 @@
+- name: Run To Do App
+  hosts: todoservers
+  remote_user: ec2-user
+      
+  vars_prompt: 
+    - name: trello_key
+      prompt: Enter Trello API key
+      private: no
+
+    - name: trello_token
+      prompt: Enter Trello API token
+
+    - name: secret_key
+      prompt: Enter secret key
+
+  tasks:
+    - name: Install Git and Python3
+      ansible.builtin.yum:
+        name:
+          - git
+          - python3
+        state: present
+      become: yes
+
+    - name: Install poetry
+      ansible.builtin.shell: curl -sSL https://install.python-poetry.org | python3 -
+      args:
+        creates: ~/.local/bin/poetry
+
+    - name: Make todo app directory
+      ansible.builtin.file:
+        path: /opt/todoapp
+        state: directory
+        mode: '0755'
+        owner: ec2-user
+      become: yes
+
+    - name: Clone app and checkout main branch
+      ansible.builtin.git: 
+        repo: 'https://github.com/jswoodburn/DevOps-Course-Starter.git'
+        dest: /opt/todoapp
+        version: master
+
+    - name: Install project dependencies
+      ansible.builtin.shell: /home/ec2-user/.local/bin/poetry install
+      args:
+        chdir: /opt/todoapp
+
+    - name: Set up environment variables
+      ansible.builtin.template:
+        src: .env.j2
+        dest: /opt/todoapp/.env
+        owner: ec2-user
+        mode: '0777'
```

Comment:
Not major, but it's worth avoiding getting into the habit of making permissions '0777' by default - everything will work but security might hate you! This file for example will have secrets in, so avoiding any "other" users from having access at all feels sensible